packages/@okta/vuepress-site/docs/guides/pwd-optional-best-practices/main/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Recommended best practices
+---
+
+A list of recommended best practices for password-optional use cases using the Embedded SDK.
+
```

Comment:
```suggestion
## Protect your admin accounts

Before you configure the passwordless experience, ensure that your admins continue to have passwords available. This ensures that users who don't have a password provisioned are not inadvertently blocked from signing in. To do this:

* Create a separate group for admins and add your admin users to that group.
* Create separate authenticator enrollment, global session, and authentication policies for this group.
* Place this group at the highest priority (at no. 1) in the authenticator enrollment policy.

```

Reply:
I've updated the guide with this section (with some amendments). Also, it's hard for me to make that mental leap of how users without passwords will get locked out if they don't create an Admin group.

Reply:
let's talk about this one.